packages/discord.js/src/managers/GuildInviteManager.js

```diff
@@ -33,7 +33,7 @@ class GuildInviteManager extends CachedManager {
   }
 
   /**
-   * Data that resolves to give an Invite object. This can be:
+   * Data that resolves to give a `GuildInvite`. This can be:
```

Comment:
We have this type defined twice (here and in DataResolver) which is also used in `Client#fetchInvite()` where it‘s not explicitly for only GuildInvite, but GroupDMInvite too. So this one here should be renamed to `GuildInviteResolvable`.